Specifying Kafka version as 1.0.0 in Note for Kafka with EH (#3474)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

| maxMessageBytes     | N | The maximum size in bytes allowed for a single Kafka message. Defaults to 1024. | `2048`
| consumeRetryInterval | N | The interval between retries when attempting to consume topics. Treats numbers without suffix as milliseconds. Defaults to 100ms. | `200ms` |
| consumeRetryEnabled | N | Disable consume retry by setting `"false"` | `"true"`, `"false"` |

> | version               | N | Kafka cluster version. Defaults to 2.0.0.0. Please note that this needs to be mandatorily set to `1.0.0` for EventHubs with Kafka. | `0.10.2.0` |

Reviewer: ```suggestion
| version               | N | Kafka cluster version. Defaults to 2.0.0.0. Note that this must be set to `1.0.0` if you using Azure EventHubs with Kafka. | `0.10.2.0` |
```

Reviewer: Should this be `1.0.0.0` ?

Author: No, `1.0.0` is correct. Infact, changed `2.0.0.0` to `2.0.0`

The `secretKeyRef` above is referencing  a [kubernetes secrets store]({{< ref kubernetes-secret-store.md >}}) to access the tls information. Visit [here]({{< ref setup-secret-store.md >}}) to learn more about how to configure a secret store component.

#### Note

> Metadata `version` needs to be mandatorily set to `1.0.0` for anyone using EventHubs with Kafka.

Reviewer: ```suggestion
The metadata `version` must be set to `1.0.0` when using Azure EventHubs with Kafka.
```
